integration/raft/channel_participation_test.go

```diff
@@ -638,6 +638,84 @@ var _ = Describe("ChannelParticipation", func() {
 			}
 		})
 	})
+
+	Describe("create system channel on empty network", func() {
```

Comment:
```suggestion
	Describe("system channel creation using the channel participation API", func() {
```

Reply:
Nit with the naming. And actually, this feels weird to have as a separate Describe. Any reason this shouldn't be part of the one above for "three node etcdraft network without a system channel".

Reply:
As you noted, I moved the test to the existing describe. 🧹